source/switching_to_live/index.html.md.erb

```diff
@@ -51,7 +51,20 @@ Choose which card types to accept by signing into the [GOV.UK Pay admin tool](ht
 
 If you use Worldpay, SmartPay or ePDQ, you must also make sure the same card types are selected in your PSP account.
 
-### If you use Worldpay
+### Block prepaid cards
+
+You can block users from paying with prepaid cards.
+
+You must register with Visa to get permission first. To register, either:
```

Comment:
I suppose this only applies if they accept Visa cards. Technically, they could choose not to. So maybe this should start, “If you accept Visa cards…”. That would also make the relationship a bit clearer (right now it almost sounds like Visa are the get to decide whether you can block prepaid cards at all).